### PR TITLE
#4949 [Manager] Toolbox state is not reset when pipeline is deleted a…

### PR DESCRIPTION
**Pull request by konekoya**
…nd created

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4949)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/Pipeline.js [hunk 1]
```diff
@@ -500,13 +499,6 @@ const Pipeline = React.forwardRef((props, ref) => {
         fromStreams: topicFromStream,
         topic: cellData,
       };
-
-      if (cellData.isShared) {
-        // If a shared topic is removed from the Paper, we should
-        // re-render the Toolbox, so the topic can be re-added
-        // into the Paper again
-        pipelineDispatch({ type: 'setToolboxKey' });
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Pipeline.js:508]: No longer needed

**Comment by jackyoh** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by jackyoh** [comment]
retry

**Comment by konekoya** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Pull request merged**